handlers/analytics_test.go

```diff
@@ -83,6 +85,7 @@ func TestHandleLog(t *testing.T) {
 						"some_field": "some value"
 					},
 					{
+						"id": "abcde12345",
```

Comment:
Perhaps worth adding a test case without an `id` present if you're worried about the backward compatible behavior?